src/utils/astropy_util.py

```diff
@@ -0,0 +1,30 @@
+from astropy.time import Time
+from astropy.coordinates import get_sun, get_moon, CartesianRepresentation
+from astropy.constants import G, M_sun, M_earth
+from typing import Tuple, Optional
+from utils.constants import BodyEnum
+from astropy.coordinates import SkyCoord
+
+
+# Heavily reference the get_ephemeris function here: https://github.com/Cislunar-Explorers/FlightSoftware/blob/master/OpticalNavigation/core/observe_functions.py
+def get_ephemeris(observeStart: float, body: BodyEnum) -> Tuple[float, float, float]:
+    # Astropy needs unix timestamp in seconds!!!
+    current_time = observeStart
+    init_au = SkyCoord(0, 0, 0, unit="m", frame="gcrs", representation_type="cartesian")
+    current_au = SkyCoord(
+        0, 0, 0, unit="m", frame="gcrs", representation_type="cartesian"
+    )
+    if body == BodyEnum.Sun:
+        init_au = get_sun(Time(observeStart, format="unix")).cartesian
```

Comment:
If you're just getting the position of the bodies, then there is no need to have an `init_au` and a `current_au`. This was done in my code because I was also calculating velocity (so I needed 2 positions). I would suggest getting rid of `init_au` and just using `current_au` as the single variable of importance